pkgs/development/libraries/ti-rpc/default.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation rec {
     sed -es"|/etc/netconfig|$out/etc/netconfig|g" -i doc/Makefile.in tirpc/netconfig.h
   '';
 
+  configureFlags = ''
```

Comment:
I am slightly worried making this unconditional, because a future maintainer might notice that it builds fine on non-cross and remove it, thus breaking the cross case.